docs/CONTRIBUTING.rst

```diff
@@ -71,24 +81,17 @@ Development environment setup
    You can then commit and push your changes as usual. Open a Pull Request on
    GitHub to submit your code to be merged into master.
 
-Checklist for pull requests
----------------------------
-
-1. All CI jobs should pass
-2. Changes or additions should have appropriate unit tests (see below)
-3. Follow the PEP8 style guide as far as possible (with caveats below).
-4. All public functions and classes should have
-   `Google-style docstrings <https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html>`_ 
-
 Code Style
 ----------
 
 We use the yapf auto-formatter with the style configured in the repository. 
-You should either set this formatter in your IDE settings, or run the following
-command before committing:
+Most IDEs allow you to configure a formatter to run automatically when you save
+a file. Alternatively, you can run the following command before commiting any
+changes:
 
 .. code-block:: bash
 
+   # Reformat all python files in the repository
    yapf -i .
```

Comment:
I think we need -ir here

Reply:
Yes, on my mac I get the following error with the command as currently written. I will make the change to this branch and then approve: I hope that works for you.